opensearch/chart/templates/tests/test-opensearch-config.yaml

```diff
@@ -0,0 +1,123 @@
+{{- if .Values.testFramework.enabled -}}
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-test
+  namespace: {{ .Release.Namespace }}
+  labels:
+    type: integration-test
+    {{- include "opensearch-cluster-local.labels" . | nindent 4 }}
+data:
+  run.sh: |-
+
+    #!/usr/bin/env bats
+
+    load "/usr/lib/bats/bats-detik/utils"
+    load "/usr/lib/bats/bats-detik/detik"
+
+    DETIK_CLIENT_NAME="kubectl"
+
+    @test "Verify successful deployment and running status of the operator pod" {
+        try "at most 12 times every 10s to get pods named 'opensearch-operator-controller-manager-[a-z0-9]+-[a-z0-9]+' and verify that '.status.phase' is 'Running'"
+    }
+
+    @test "Verify successful deployment of the operator service" {
+        verify "there is 1 service named '^opensearch-operator-controller-manager-metrics-service$'"
```

Comment:
This regex is a literal match. So there must be a service which always has this exact name. Is that the case? What happens on ns changes? Plugin-metadata-name changes?

Reply:
Good point. I removed the standard fullnameoverride and rely on the release namespace.